packages/credentials/src/V1/KiltCredentialV1.ts

```diff
@@ -327,15 +327,77 @@ export function fromInput({
 
 const cachingCTypeLoader = newCachingCTypeLoader()
 
+// Check recursively if a value has references
+const hasRef = (value: unknown): boolean => {
+  if (typeof value !== 'object' || value === null) {
+    return false
+  }
+
+  if ('$ref' in (value as Record<string, unknown>)) {
+    return true
+  }
+
+  if (Array.isArray(value)) {
+    return value.some((item) => hasRef(item))
+  }
+
+  return Object.values(value as Record<string, unknown>).some((v) => hasRef(v))
+}
```

Comment:
you're not using this function any more, are you?

Reply:
Yes forget to remove, fixed at: https://github.com/KILTprotocol/sdk-js/pull/919/commits/bbfced24e1897748e57f40ddc5fa06267ed4962c